modules/RTC/JitsiLocalTrack.js

```diff
@@ -20,9 +20,11 @@ import RTCEvents from '../../service/RTC/RTCEvents';
 import VideoType from '../../service/RTC/VideoType';
 import {
     TRACK_UNMUTED,
-    createNoDataFromSourceEvent
+    createNoDataFromSourceEvent,
+    TRACK_VIDEOBLUR
 } from '../../service/statistics/AnalyticsEvents';
 import Statistics from '../statistics/statistics';
+import JitsiStreamBlurEffect from './JitsiStreamBlurEffect';
```

Comment:
Could you please leave an empty line? This way it will be consistent with jitsi-meet imports...